src/supplemental/nanolib/blf/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+if(NNG_ENABLE_BLF)
+    nng_sources(blf.cc)
+    find_package(Vector_BLF REQUIRED)
+    nng_include_directories(${Vector_BLF_INCLUDE_DIRS})
+    nng_link_libraries(${Vector_BLF_LIBRARIES})
```

Comment:
what if I need static link?

Reply:
If your library is static, this will only set a static link. If there are both shared and static libraries, we may need to set some options.